src/overlays/actors/ovl_En_Hy/z_en_hy.h

```diff
@@ -40,25 +63,25 @@ typedef struct EnHy {
     /* 0x0194 */ char unk_194; // unused
     /* 0x0195 */ u8 pathReverse;
     /* 0x0196 */ s8 objBankIndexHead;
-    /* 0x0197 */ s8 objBankIndexSkel2; // 7 < limb < 15 (upper part?) (always same as objBankIndexSkel1)
-    /* 0x0198 */ s8 objBankIndexSkel1; // sets the object used when drawing the skeleton for limb <= 7 (lower part?)
+    /* 0x0197 */ s8 objBankIndexUpperSkel; // upper body limbs object, 7 < limb < 15 (always same as objBankIndexLowerSkel),
+    /* 0x0198 */ s8 objBankIndexLowerSkel; // lower body limbs object, limb <= 7
     /* 0x0199 */ s8 objBankIndexOsAnime;
     /* 0x019C */ ColliderCylinder collider;
     /* 0x01E8 */ NpcInteractInfo interactInfo;
     /* 0x0210 */ Path* path;
     /* 0x0214 */ s8 waypoint;
-    /* 0x0215 */ s8 unk_215;
+    /* 0x0215 */ s8 playedSfx;
     /* 0x0216 */ char unk_216[2]; // unused
     /* 0x0218 */ s16 curEyeIndex;
     /* 0x021A */ s16 nextEyeIndexTimer;
-    /* 0x021C */ s16 unk_21C[16]; // bodyWiggleY ?
-    /* 0x023C */ s16 unk_23C[16]; // bodyWiggleZ ?
+    /* 0x021C */ s16 fidgetTableY[16]; // bodyWiggleY ?
+    /* 0x023C */ s16 fidgetTableZ[16]; // bodyWiggleZ ?
```

Comment:
Im not 100% sure about calling this a "fidget table" in general. I am aware that mm just recently moved to this terminology, but I wasnt around at the time to take part in the conversation.

Guess its ok for now, but will prob want to discuss it on a broader scale at some point in the future.
Can prob remove the the comments on them too

Reply:
I'm always down to discuss names, so if we do have other suggestions please just let me know. Removed the comments at least.